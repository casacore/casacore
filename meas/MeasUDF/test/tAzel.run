#!/bin/sh

export LD_LIBRARY_PATH=../..:${LD_LIBRARY_PATH}
export DYLD_LIBRARY_PATH=../..:${DYLD_LIBRARY_PATH}

$casa_checktool ./azel jupiter dwl "2016-06-28 19:54"
