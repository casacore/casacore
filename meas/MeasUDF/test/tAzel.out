az: -116.371 deg
el: 26.2849 deg
